File: runelite-client/src/main/java/net/runelite/client/plugins/microbot/vorkath/PRAYER_POTION.java
package net.runelite.client.plugins.microbot.vorkath;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.val;

@Getter
@RequiredArgsConstructor
public enum PRAYER_POTION {
    PRAYER("prayer potion"),
    SUPER_RESTORE("Super restore"),
    MOONLIGHT_MOTH_MIX("Moonlight moth mix");

    private final String potionName;

    @Override
    public String toString() { return potionName; }
}

File: runelite-client/src/main/java/net/runelite/client/plugins/jrPlugins/autoVorkath/enums/PRAYER_POTION.kt
/*
 * Copyright (c) 2024. By Jrod7938
 *
 */
package net.runelite.client.plugins.jrPlugins.autoVorkath.enums

import lombok.Getter
import lombok.RequiredArgsConstructor

@Getter
@RequiredArgsConstructor
enum class PRAYER_POTION(private val potionName: String) {
    PRAYER("Prayer potion"),
    SUPER_RESTORE("Super restore");

    override fun toString(): String {
        return potionName
    }
}